govulndb/govulndb.go

```diff
@@ -0,0 +1,69 @@
+package govulndb
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"golang.org/x/xerrors"
+
+	pb "gopkg.in/cheggaaa/pb.v1"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	baseURL     = "https://storage.googleapis.com/go-vulndb"
+	goVulnDBDir = "go"
+	concurrency = 5
+	wait        = 0
+	retry       = 5
+)
+
+func Update() error {
+	log.Println("Fetching GoVulnDB data...")
+
+	b, err := utils.FetchURL(baseURL+"/index.json", "", retry)
```

Comment:
Ideally, we should use `path.Join`.
https://stackoverflow.com/questions/34668012/combine-url-paths-with-path-join/34668130